Singer.API/Services/UserService.cs

```diff
@@ -86,23 +81,16 @@ public override async Task DeleteAsync(Guid id)
 
       public override async Task<SearchResults<TUserDTO>> GetAsync(
          string filter = null,
-         Expression<Func<TUserEntity, TUserDTO>> projector = null,
          Expression<Func<TUserDTO, object>> orderer = null,
          ListSortDirection sortDirection = ListSortDirection.Ascending,
          int pageIndex = 0,
          int entitiesPerPage = 15)
       {
-         // set the projector if it is null
-         if (projector == null)
-            projector = EntityToDTOProjector;
-
          // return the paged results
-         return await Queryable  // TODO: Unsure if this is OK, we used to have DBSet here, but that doesn't have the linked entities resolved :(
```

Comment:
You removed the //TODO comment, does that mean it is Ok to use `Queryable` instead of `DbSet` here?